### PR TITLE
ES-863989-Add LaTeX to Math and Math to LaTeX UG changes in java

### PR DESCRIPTION
**Pull request by AntonihilSahayaraj**
In this PR, i have added the LaTex and Supported LaTex pages under the Working with Mathematical Equation page

### CODE REVIEW

File: java-file-formats/word-library/Working-with-LaTeX.md [hunk 1]
```diff
@@ -0,0 +1,1332 @@
...
+documentation: UG
+---
+
+# Create Equation using LaTeX
+The Java Word library allows to create mathematical equation in Word document using **LaTeX**.
+
+## Accent
+
+Add **accent** equation to a Word document using the LaTeX through appendMath API.
+
```
Comment (LokeshBaskar) [inline java-file-formats/word-library/Working-with-LaTeX.md:15]: Highlight the API name as like this(\`) `appendMath`

File: java-file-formats/word-library/Working-with-LaTeX.md [hunk 2]
```diff
@@ -0,0 +1,1332 @@
...
+The following code example illustrates how to create accent equation using LaTeX in Word document.
+
+{% tabs %}  
+
+{% highlight JAVA %}
+
+//Creates a new Word document
+WordDocument document = new WordDocument();
```
Comment (LokeshBaskar) [inline java-file-formats/word-library/Working-with-LaTeX.md:23]: Please add the period(.) at end of the comment.

File: java-file-formats/word-library/Working-with-LaTeX.md [hunk 3]
```diff
@@ -0,0 +1,1332 @@
...
+</tr>
+<tr>
+<td>2.</td>
+<td><img src="WorkingwithMathematicalEquation_images/Fraction2.png" alt="Fraction equation"></td>
+<td>\sfrac{dy}{dx}</td>
+</tr>
+<tr>
+<td>3.</td>
+																																 
+
```
Comment (LokeshBaskar) [inline java-file-formats/word-library/Working-with-LaTeX.md:517]: Please remove this empty lines

**Review by LokeshBaskar — CHANGES_REQUESTED** [review]

File: java-file-formats-toc.html [hunk 4]
```diff
@@ -160,6 +160,10 @@
             </li>
             <li>
                 <a href="/java-file-formats/word-library/working-with-mathematical-equation">Working with Mathematical Equation</a>
+				<ul>
+					<li><a href="/java-file-formats/word-library/Working-with-LaTeX">LaTeX</a></li>
+					<li><a href="/java-file-formats/word-library/Supported-LaTeX.md">Supported LaTeX</a></li>
```
[the remark this answers is not in the document]
Comment (AntonihilSahayaraj) [reply java-file-formats-toc.html]: Cased the md file names to the lower and removed the .md file extension

File: java-file-formats/word-library/Supported-LaTeX.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+
+## Arrows
+
+The following image demonstrates the LaTeX supported in Java Word library to create arrow symbols.
+
+![Create Arrow Symbols in LaTeX](WorkingwithMathematicalEquation_images/ArrowsInLaTex_1.png)
+![Create Arrow Symbols in LaTeX](WorkingwithMathematicalEquation_images/ArrowsInLaTex_2.png)
+![Create Arrow Symbols in LaTeX](WorkingwithMathematicalEquation_images/ArrowsInLaTex_3.png)
+![Create Arrow Symbols in LaTeX](WorkingwithMathematicalEquation_images/ArrowsInLaTex_4.png)
+![Create Arrow Symbols in LaTeX](WorkingwithMathematicalEquation_images/ArrowsInLaTex_5.png)
```
[the remark this answers is not in the document]
Comment (AntonihilSahayaraj) [reply java-file-formats/word-library/Supported-LaTeX.md]: added the required images in the folder.

File: java-file-formats/word-library/Supported-LaTeX.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+
+## Basic Math
+
+The following image demonstrates the LaTeX supported in Java Word library to create basic math symbols.
+
+![Create Basic Math Symbols in LaTeX](WorkingwithMathematicalEquation_images/BasicMath_1.png)
+![Create Basic Math Symbols in LaTeX](WorkingwithMathematicalEquation_images/BasicMath_2.png)
+![Create Basic Math Symbols in LaTeX](WorkingwithMathematicalEquation_images/BasicMath_3.png)
+![Create Basic Math Symbols in LaTeX](WorkingwithMathematicalEquation_images/BasicMath_4.png)
+
```
[the remark this answers is not in the document]
Comment (AntonihilSahayaraj) [reply java-file-formats/word-library/Supported-LaTeX.md]: added the required images in the folder.

File: java-file-formats/word-library/Working-with-LaTeX.md (hunk 1 above)
Comment (AntonihilSahayaraj) [reply java-file-formats/word-library/Working-with-LaTeX.md]: Highlighted the API name as like appendMath() in places

File: java-file-formats/word-library/Working-with-LaTeX.md (hunk 3 above)
Comment (AntonihilSahayaraj) [reply java-file-formats/word-library/Working-with-LaTeX.md]: Remove this empty lines

**Review by LokeshBaskar — APPROVED** [review]
No concerns

**Pull request merged**